tempest/src/test/kotlin/app/cash/tempest/DynamoDbViewTest.kt

```diff
@@ -107,13 +108,13 @@ class DynamoDbViewTest {
     assertThat(actualPlaylistInfoV2).isEqualTo(playlistInfoV2)
 
     // This fails because playlist_size is already 1.
-    assertThatExceptionOfType(ConditionalCheckFailedException::class.java)
-      .isThrownBy {
-        musicTable.playlistInfo.save(
-          playlistInfoV2,
-          ifPlaylistVersionIs(playlistInfoV1.playlist_version)
-        )
-      }
+    val exception = assertThrows<AmazonServiceException> {
```

Comment:
Unfortunate consequence of a bug in the DynamoDb Local client where it throws a different exception than what would be thrown in a real-world scenario.